ethcore/src/evm/wasm/call_args.rs

```diff
@@ -0,0 +1,64 @@
+// Copyright 2015-2017 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+
+// Parity is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Wasm evm call arguments helper
+
+use util::{U256, H160};
+
+/// Input part of the wasm call descriptor
+pub struct CallArgs {
+    /// Receiver of the transaction
+    pub address: [u8; 20],
+
+    /// Sender of the transaction
+    pub sender: [u8; 20],
+
+    /// Original transaction initiator
+    pub origin: [u8; 20],
+
+    /// Transfer value
+    pub value: [u8; 32],
+
+    /// Reserved space / alignment to 256 bytes
+    _reserved: [u8; 164],
```

Comment:
sure, on second thought, it is not required
I thought it might be useful for contracts to be compatible when the header changes (extends or shrinks), but this can be achieved by other more consensus-friendly means (versioned header)